cloud: create the TiDB Cloud partner doc (#18785)
pingcap/docs: tidb-cloud/cppo-customer.md

---

> title: AWS Channel Partner Private Offer Customer

Author: ```suggestion
title: AWS Channel Partner Private Offer (CPPO) Customer
```

---

> title: AWS Channel Partner Private Offer (CPPO) Customer

Reviewer: ```suggestion
title: Reseller's Customer
```

> summary: Learn how to become an AWS Channel Partner Private Offer (CPPO) customer.

Reviewer: ```suggestion
summary: Learn how to become a reseller's customer.
```

---

> # AWS Channel Partner Private Offer (CPPO) Customer

Reviewer: ```suggestion
# Reseller's Customer
```

> A reseller's customer subscribes to TiDB Cloud services through a Channel Partner in the AWS Marketplace.

Reviewer: ```suggestion
A reseller's customer subscribes to TiDB Cloud services through a reseller in the AWS Marketplace.
```

> An AWS Channel Partner Private Offer (CPPO) customer subscribes to TiDB Cloud services through a Channel Partner in the AWS Marketplace.

Reviewer: ```suggestion
A Reseller's customer subscribes to TiDB Cloud services through a Channel Partner in the AWS Marketplace.
```

> Unlike direct TiDB Cloud customers, a reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.

Reviewer 2: ```suggestion
Unlike direct TiDB Cloud customers, a reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS Marketplace Channel Partner Private Offer (CPPO) and direct TiDB Cloud customers.
```

> Unlike direct TiDB Cloud customers, AWS CPPO customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.

Reviewer: ```suggestion
Unlike direct TiDB Cloud customers, reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.
```

This document outlines how to check both future and historical bills.

## Migrate from a direct TiDB Cloud account to a reseller's customer account

> **Tip:**
>

> > Direct customers are the end customers who purchase TiDB Cloud and pay invoices directly from PingCAP.

Reviewer 2: ```suggestion
> Direct customers are the end customers who directly purchase TiDB Cloud from and pay invoices to PingCAP.
```

> ## Migrate from a direct TiDB Cloud account to an AWS CPPO customer account

Reviewer: ```suggestion
## Migrate from a direct TiDB Cloud account to a reseller's customer account
```

If you are currently a direct customer with a TiDB Cloud account, you can ask your reseller to migrate your account to a reseller's customer account.

> The migration will take effect on the first of a future month. Discuss with your reseller to determine the specific effective date.

Reviewer 2: ```suggestion
The migration will take effect in the next month. Discuss with your reseller to determine the specific effective date.
```

> The migration will take effect on the 1st of a future month. Discuss with your reseller to determine the specific effective date.

Reviewer 2: ```suggestion
The migration will take effect on the first day of a future month. Discuss with your reseller to determine the specific effective date.
```

> The migration will take effect on the 1st of a future month. Discuss with your reseller to determine the specific effective date.

Author: ```suggestion
The migration will take effect on the first day of a future month. Discuss with your reseller to determine the specific effective date.
```

On the effective day of migration, you will receive an email notification.

## View your future bills

> As a reseller's customer, your invoices will be processed and paid through the AWS Marketplace.

Reviewer 2: Made some edits because the subject of "As a ..." is "you".
```suggestion
As a reseller's customer, you will have your invoices processed and paid through the AWS Marketplace.
```

> If you are currently a direct customer with a TiDB Cloud account, you can ask your AWS CPPO to migrate your account to an AWS CPPO customer account.
> 
> The migration will take effect on the first of a future month. Discuss with your AWS CPPO partner to determine the specific effective date.
> 
> On the effective day of migration, you will receive an email notification.

Reviewer: ```suggestion
If you are currently a direct customer with a TiDB Cloud account, you can ask your reseller to migrate your account to an reseller's customer account.

The migration will take effect on the first of a future month. Discuss with your reseller to determine the specific effective date.

On the effective day of migration, you will receive an email notification.
```

> ## View your history bills

Reviewer 2: To keep wording consistent with that in line 12.
```suggestion
## View your historical bills
```

> If you have migrated from a direct TiDB Cloud account to an AWS CPPO account, you can still access your billing history before the migration. To do so, go to **Billing** > **Bills** > **History** in the TiDB Cloud console.

Reviewer: ```suggestion
If you have migrated from a direct TiDB Cloud account to a reseller's customer account, you can still access your billing history before the migration. To do so, go to **Billing** > **Bills** > **History** in the TiDB Cloud console.
```
